MapboxGeocoderTests/BatchGeocodingTests.swift

```diff
@@ -0,0 +1,386 @@
+            
+            expectation.fulfill()
+        }
+        XCTAssertNotNil(task)
+        
+        waitForExpectations(timeout: 1) { (error) in
+            XCTAssertNil(error, "Error: \(error!)")
+            XCTAssertEqual(task.state, URLSessionTask.State.completed)
+        }
+    }
+    
+    func testValidReverseMultipleBatchGeocode() {
+        let expectation = self.expectation(description: "forward batch geocode with multiple queries should return results")
+        
+        _ = stub(condition: isHost("api.mapbox.com")
+            && isPath("/geocoding/v5/mapbox.places-permanent/-77.01073,38.88887;-77.01073,38.88887;-77.01073,38.88887.json")
+            && containsQueryParams(["access_token": BogusToken])) { _ in
+                let path = Bundle(for: type(of: self)).path(forResource: "permanent_reverse_multiple_valid", ofType: "json")
+                return OHHTTPStubsResponse(fileAtPath: path!, statusCode: 200, headers: ["Content-Type": "application/vnd.geo+json"])
+        }
+        
+        let geocoder = Geocoder(accessToken: BogusToken)
+        
+        let queries = [
+            CLLocationCoordinate2DMake(38.88887, -77.01073),
+            CLLocationCoordinate2DMake(38.88887, -77.01073),
+            CLLocationCoordinate2DMake(38.88887, -77.01073)
+        ]
+        
+        let options = ReverseBatchGeocodeOptions(coordinates: queries)
+
+        let task = geocoder.batchGeocode(options) { (placemarks, attribution, error) in
+            
+            let queries = placemarks!
+            let attribution = attribution![0]
+            
+            XCTAssertEqual(queries.count, 3, "forward batch geocode should have 3 queries")
+            
+            for result in queries {
+                XCTAssertEqual(result.count, 6, "each forward batch geocode query should have 6 found results")
+                XCTAssertEqual(attribution, "© 2017 Mapbox and its suppliers. All rights reserved. Use of this data is subject to the Mapbox Terms of Service. (https://www.mapbox.com/about/maps/)")
+            }
+            
+            let sampleResult = queries[0][0]
+            
+            XCTAssertEqual(sampleResult.qualifiedName, "South Capitol Circle Southwest, Washington, District of Columbia 20002, United States")
+            
+            expectation.fulfill()
+        }
+        XCTAssertNotNil(task)
+        
+        waitForExpectations(timeout: 1) { (error) in
+            XCTAssertNil(error, "Error: \(error!)")
+            XCTAssertEqual(task.state, URLSessionTask.State.completed)
+        }
+    }
+    
+    func testNoResultsReverseSingleBatchGeocode() {
+        _ = stub(condition: isHost("api.mapbox.com")
+            
+           // && isPath("/geocoding/v5/mapbox.places-permanent/100,100.json")
+              // The above line is causing the test fo fail for unknown reasons
```

Comment:
@1ec5 I've added the reverse batch geocoding tests but I'm seeing the weird behavior with OHTTPS `isPath` again - I get test failures if I include the line above for both reverse batch geocoding tests that return no results. The tests pass if I remove this line though 🤕

Reply:
Oh! Yeah, it looks like `urlForGeocoding(_:)` is turning this into `100.00000,100.00000`.